omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,57 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',function ($scope,TagService,ngDialog) {
+    $scope.init = function(personUuid) {
+            $scope.thisPersonUuid = personUuid;
+          $scope.loadTags =function () {
+              var payload = {
+                  objectType: 'org.openmrs.Person',
+                  objectUuid: personUuid
+              };
+              TagService.getTags(payload).then(function (res) {
+                  $scope.tags = res;
+              });
+          }
+         $scope.loadTags();
+    }
+
+        /**
+         * Delete Tag Dialog
+         */
+        $scope.showDialog = function(tag) {
+            ngDialog.openConfirm({
+                showClose: false,
+                closeByEscape: true,
+                closeByDocument: true,
+                data: angular.toJson({
```

Comment:
Why do you need to call toJson()? Isn't it enough to just assign a JS object as data?